static/css/ghe-loading.css

```diff
@@ -0,0 +1,45 @@
+.gheLoading___body {
+    background: #fff;
+}
+
+.gheLoading___container {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    justify-content: center;
+    margin: auto;
+    max-width: 648px;
+}
+
+.gheLoading___header {
+    font-size: 1.5rem;
+}
+
+.gheLoading___imgDivider {
+    font-size: 2.8rem;
+    color: #6B778C;
+}
+
+.gheLoading___imgContainer {
+    margin: 4.688em 0 4.063em;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    min-width: 375px;
+}
+
+.gheLoading___jiraImg {
+    height: 80px;
+}
+
+.gheLoading___GitHubImg {
+    height: 60px;
+}
+
+.gheLoading___message {
+    text-align: center;
+    max-width: 364px;
```

Comment:
Why do we need a max-width here? Normally you'd just set a max-width on the outer container so that all child elements don't extend beyond that point.

Reply:
This content has a small width than the parent, so used `max-width` here so that the content doesn't exceed beyond this. Can use `width` too, but don't really see much difference in this case?

Reply:
not sure we need any width for a paragram element. The child would never exceed the parent width unless you had set text overflow to none;

Reply:
The width was added to match the design to make it two lines.